### PR TITLE
Update automated email (fixes #382)

### LINKED ISSUE #382: Email needs formatting and updating
# Current Email
Welcome to Operation Code! You've just joined a community of 3000+ coders, who are all looking to further their skills and careers in software development!
We are a decentralized organization, and we work both [online](https://community.operationcode.org/) and in [local chapters](https://www.meetup.com/pro/operationcode/) across the country. Please introduce yourself on our [forums](https://community.operationcode.org/c/introductions), and come join one of our [meetups in your local area](https://www.meetup.com/pro/operationcode/)! 
Also, be sure to check your email for an invite to the Operation Code Slack community ([https://operation-code.slack.com](https://operation-code.slack.com)).
Slack is where we communicate online in real-time to ask questions, collaborate, and share information. Once in Slack (you'll probably want to download the app), introduce yourself in the #general channel. An example could be: 
"Hi all. My name is David. I'm from Washington and I'm interested in web development!" Please read our [Code of Conduct](http://op.co.de/code-of-conduct), as well.

The Operation Code website uses React, a JavaScript framework, and Ruby on Rails. If you want to dive in and help, you can get started by reading our [docs](https://github.com/OperationCode/operationcode/blob/master/CONTRIBUTING.md), or asking around in the #oc-projects Slack channel. If you're interested in our mentorship program, fill out our application form as either a [mentor](http://op.co.de/mentor-enrollment) or [mentee](http://op.co.de/mentor-request), also!

Again, welcome aboard, and don't forget to follow and share Operation Code on [Twitter](https://twitter.com/operation_code) and [Facebook](https://facebook.com/operationcode.org).

Cheers, and keep coding,
the Operation Code team :)
[@operation_code](http://twitter.com/operation_code)

P.S. - If you do not receive an invite from Slack within 2 hours, please [email our staff](mailto:staff@operationcode.org), or ping us on Twitter at @operation_code.

# Proposed Email
Welcome to Operation Code! 

Be on the lookout for an email invite to the Operation Code Slack community ([https://operation-code.slack.com](https://operation-code.slack.com)). It's a pivotal aspect of our community! If you've never used Slack before, consider watching [this instructional video on using the application](https://www.youtube.com/watch?v=s69uoRkm0WE&t=56s).

You've just joined a community of 3000+ coders, who are all looking to further their skills and careers in software development! We are a decentralized organization, and we work both online (via Slack) and in [local chapters](https://www.meetup.com/pro/operationcode/) across the country. We highly recommend giving yourself an introduction in the #general channel on Slack, and to sign up to go to your next local meetup!

Please read our [Code of Conduct](http://op.co.de/code-of-conduct).

Are you interested in helping Operation Code's website through code or design? We're open source and love onboarding new developers! [Check this out](https://github.com/OperationCode/START_HERE) for more information, and join the #oc-projects channel on Slack - feel free to ask about anything.

Again, welcome aboard, and don't forget to follow and share Operation Code on Twitter [@operation_code](https://twitter.com/operation_code) and [Facebook](https://facebook.com/operationcode.org).

Cheers,
- Operation Code

P.S. If you do not receive an invite from Slack within 2 hours, please [email our staff](mailto:staff@operationcode.org), or ping us on Twitter at [@operation_code](https://twitter.com/operation_code).

### PR DESCRIPTION
**Pull request by hollomancer**
# Description of changes
Copy updates to the automated email for readability and onboarding. Removes outdated links.

# Issue Resolved
Fixes #382

### CODE REVIEW

File: app/views/user_mailer/welcome.html.erb [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
-  Also, be sure to check your email for an invite to the Operation Code Slack community (<a href="https://operation-code.slack.com">https://operation-code.slack.com</a>).<br />
-  Slack is where we communicate online in real-time to ask questions, collaborate, and share information. Once in Slack (you'll probably want to download the app), introduce yourself in the #general channel. An example could be: <br />
+  You've just joined a community of 3000+ software developers, veterans, and military spouses, who are all looking to further their skills and careers in tech! We are a decentralized organization, and we work both online (via Slack) and in <a href="https://www.meetup.com/pro/operationcode/">local chapters across the country.</a> We highly recommend giving yourself an introduction in the #general channel on Slack, and to sign up to go to your next local meetup! An example could be: <br />
   "Hi all. My name is David. I'm from Washington and I'm interested in web development!"
   Please read our <a href="http://op.co.de/code-of-conduct">Code of Conduct</a>, as well.
```
Comment (wimo7083) [inline app/views/user_mailer/welcome.html.erb]: Is this the right code of conduct?
https://github.com/OperationCode/operationcode_docs/blob/master/community/code_of_conduct.md

File: app/views/user_mailer/welcome.html.erb [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
 </p>
 
 <p>
-  The Operation Code website uses React, a JavaScript framework, and Ruby on Rails. If you want to dive in and help, you can get started by reading our <a href="https://github.com/OperationCode/operationcode/blob/master/CONTRIBUTING.md">docs</a>, or asking around in the #oc-projects Slack channel.
-  If you're interested in our mentorship program, fill out our application form as either a <a href="http://op.co.de/mentor-enrollment">mentor</a> or <a href="http://op.co.de/mentor-request">mentee</a>, also!
+  Are you interested in helping Operation Code's website through code or design, and building your experience and portfolio? We're open source and love onboarding new developers! <a href="https://github.com/OperationCode/START_HERE">Read more here</a>, and join the #oc-projects channel on Slack - feel free to ask about anything.
```
Comment (wimo7083) [inline app/views/user_mailer/welcome.html.erb]: Let's use 

`<a href="https://operation-code.slack.com/messages/C04FASBU9">#oc-projects</a>`

Comment (nellshamrell) [reply app/views/user_mailer/welcome.html.erb]: I kind of like having it as join the #oc-projects channel on Slack - as the direct link might be a little funky depending on where/how someone is running Slack.

Comment (wimo7083) [reply app/views/user_mailer/welcome.html.erb]: That works with me :)

**Comment by wimo7083** [comment]
Travis build isn't reporting here that it worked. But the status is actually complete on the travis site:
image

**Pull request merged**